mantidproject/mantid
# Indirect Simulations SimulatedDensityOfStates - Ensure that the Bin width actually sets the bins to the expected width

The `BinWidth` property in SimulatedDensityOfStates should no longer change the x range in the data and instead should rebin the data to the correct bin width

**To test:**

The following script should produce 2 workspace group. Ensure that the data in each of them is roughly comparable and the bin width in the second workspace is 5.

```
SDOS = SimulatedDensityOfStates(PHONONFile='squaricn.phonon', spectrumType='DOS', Ions='H,C,O')
Rebined_SDOS = SimulatedDensityOfStates(PHONONFile='squaricn.phonon', spectrumType='DOS', BinWidth=5, Ions='H,C,O')
```
- Ensure that the X range for the data is not altered and only the binning is changed

Fixes #15629.

**[Release notes](https://github.com/mantidproject/mantid/blob/15629_SDOS_scale_with_bin_width/docs/source/release/v3.7.0/indirect_inelastic.rst)**

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
